cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+   "execution_count": null,
+   "id": "06915543",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Calculate CROWDLAB performance against the ground truth for each epoch\n",
+    "for i in range(itter):\n",
+    "    pred_probs = model_results[i]['pred_probs']\n",
+    "    consensus_labels_in = model_results[i]['consensus_labels_in']\n",
+    "    consensus_labels_out = model_results[i]['consensus_labels_out']\n",
+    "\n",
+    "    acc_model_gtruth = (pred_probs.argmax(axis=1) == true_labels).mean()\n",
+    "    acc_consensus_gtruth = (consensus_labels_in == true_labels).mean()\n",
+    "\n",
+    "    results = {\n",
+    "        \"consensus_gtruth_accuracy\": acc_consensus_gtruth, # consensus labels accuracy \n",
+    "        \"model_gtruth_accuracy\": acc_model_gtruth,         # model label accuracy\n",
+    "    }\n",
+    "    \n",
+    "    model_results[i].update(results)\n",
+    "\n",
+    "# Calculate accuracy of final CROWDLAB generated consensus label\n",
+    "final_consensus_labels = model_results[itter-1][\"consensus_labels_out\"]\n",
+    "final_consensus_accuracy = (final_consensus_labels == true_labels).mean()\n",
+    "\n",
+    "print(\"Final consensus label accuracy (vs ground truth): \", final_consensus_accuracy)\n",
+    "print(\"Final model predictions accuracy (vs ground truth): \", model_results[itter-1][\"model_gtruth_accuracy\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03bb6b74",
+   "metadata": {},
+   "source": [
+    "Finally lets plot the results over multiple epochs and observe the improvement. Here epoch -1 corresponds to the initial `consensus_labels` generated using majority vote without the help of a model. Generating consensus with majority vote is the most common practice and therefore makes a great baseline to compare CROWDLAB against.\n",
+    "\n",
+    "As you can see, the accuracy of the `consensus_labels` improves over several itterations of model training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b97270d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "consensus_gtruth_accuracy = [model_results[i][\"consensus_gtruth_accuracy\"] \n",
+    "                             for i in range(itter)] + [final_consensus_accuracy]\n",
+    "\n",
+    "model_gtruth_accuracy = [model_results[i][\"model_gtruth_accuracy\"] for i in range(itter)]\n",
+    "\n",
+    "\n",
+    "# plot prc\n",
+    "plt.rcParams[\"figure.figsize\"] = (17,6)\n",
+    "plt.show()\n",
+    "\n",
+    "plt.subplot(1, 2, 1)\n",
+    "plt.plot(range(-1, itter),consensus_gtruth_accuracy)\n",
+    "plt.xlabel(\"Epochs\", fontsize=14)\n",
+    "plt.ylabel(\"Consensus Label Accuracy\", fontsize=14)\n",
+    "plt.title(\"Consensus Label vs Ground Truth Accuracy\", fontsize=14, fontweight=\"bold\")\n",
+    "\n",
+    "plt.subplot(1, 2, 2)\n",
+    "plt.plot(range(0, itter), model_gtruth_accuracy)\n",
+    "plt.xlabel(\"Epochs\", fontsize=14)\n",
```

Comment:
This should definitely not be called Epochs, see comment above "Confusing language alert"